dockstore-webservice/src/main/resources/migrations.1.8.0.xml

```diff
@@ -80,4 +80,18 @@
         <addUniqueConstraint columnNames="imageid" constraintName="uk_imageid" tableName="entry_version_image"/>
         <addForeignKeyConstraint baseColumnNames="imageid" baseTableName="entry_version_image" constraintName="fk_entryVersionImage" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="image"/>
     </changeSet>
+    <changeSet author="gluu (generated)" id="addVersionMetadata">
+        <addColumn tableName="version_metadata">
+            <column name="author" type="varchar(255 BYTE)"/>
```

Comment:
I usually prefer varchar(255) so that I don't need to worry about multi-byte characters.
https://blog.lboro.ac.uk/middleware/blog/databases/identifying-multibyte-utf-8-characters-in-postgresql

Same with the following